public/app/plugins/panel/stat/module.tsx

```diff
@@ -17,22 +17,36 @@ export const plugin = new PanelPlugin<Options>(StatPanel)
     addOrientationOption(builder, mainCategory);
     commonOptionsBuilder.addTextSizeOptions(builder);
 
-    builder.addSelect({
-      path: 'textMode',
-      name: 'Text mode',
-      description: 'Control if name and value is displayed or just name',
-      category: mainCategory,
-      settings: {
-        options: [
-          { value: BigValueTextMode.Auto, label: 'Auto' },
-          { value: BigValueTextMode.Value, label: 'Value' },
-          { value: BigValueTextMode.ValueAndName, label: 'Value and name' },
-          { value: BigValueTextMode.Name, label: 'Name' },
-          { value: BigValueTextMode.None, label: 'None' },
-        ],
-      },
-      defaultValue: defaultOptions.textMode,
-    });
+    builder
+      .addSelect({
+        path: 'textMode',
+        name: 'Text mode',
+        description: 'Control if name and value is displayed or just name',
```

Comment:
```suggestion
        description: 'Control if value, name, or both are displayed',
```

Reply:
This suggestion is a different meaning 🤔 `textMode` only applies for name + value, name, but not just value 😬